### PR TITLE
[releng] Set version to 2.20.0

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.activities/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.eclipse.xtext</groupId>
 		<artifactId>org.eclipse.xtext.tycho.plugin.parent</artifactId>
-		<version>2.19.0-SNAPSHOT</version>
+		<version>2.20.0-SNAPSHOT</version>
```
Comment (szarnekow) [inline org.eclipse.xtext.activities/pom.xml]: Unrelated question: Given that we have a relative path to the parent - shouldn't it be possible to maintain the version only there?

Comment (cdietrich) [reply org.eclipse.xtext.activities/pom.xml]: afaik the parent version is needed

Comment (szarnekow) [reply org.eclipse.xtext.activities/pom.xml]: Hmm that's bad.

Looks like the fancy "new" thing `${revision}` is also not very nice (https://maven.apache.org/maven-ci-friendly.html)

Comment (kthoms) [reply org.eclipse.xtext.activities/pom.xml]: That's one of the drawbacks of Maven, unfortunately. I also wish that the parent's version could be obsolete, would make many things easier.
But when you think of deployed POMs it becomes obvious: Suppose you try to retrieve an artifact which inherits the parent's version and uses the `project.parent.version` variable. When there is only a relative path and you download the artifact, it is unclear which artifact should be downloaded as the parent POM.

Comment (szarnekow) [reply org.eclipse.xtext.activities/pom.xml]: My assumption was, that the version number could be added / would be added automatically before / during signing / deployment.

**Pull request merged**